modules/installation-aws-about-government-region.adoc

```diff
@@ -39,13 +30,14 @@ The following AWS GovCloud partitions are supported:
 endif::aws-gov[]
 
 ifdef::aws-secret[]
-The following AWS Top Secret Region partition is supported:
+The following AWS secret partitions are supported:
 
-* `us-iso-east-1`
+* `us-isob-east-1` (SC2S)
+* `us-iso-east-1` (C2S)
```

Comment:
Not sure why we use word `partition` in the first version, but I suggest to use `region`, to keep it the same as https://github.com/openshift/openshift-docs/pull/44998/files#diff-13f79a93986014af34593fb13bacd33e857ce0b7493678da49aa73cb6c6c75b1R53-R56

```
The following AWS secret regions are supported:

* `us-isob-east-1` Secret Commercial Cloud Services (SC2S)
* `us-iso-east-1` Commercial Cloud Services (C2S)
```